src/commonControls/listControl/ListControlAPL.ts

```diff
@@ -0,0 +1,112 @@
+import i18next from 'i18next';
+import { RequestChangedValueByListAct } from '../..';
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+interface APLListItem {
+    primaryText: string;
+}
+
+export namespace ListControlAPLPropsBuiltIns {
+    /*
+     * For information about the TextListTemplate, see following doc:
+     * https://developer.amazon.com/en-US/docs/alexa/alexa-presentation-language/apl-alexa-text-list-layout.html
+     */
+    export const TextList = {
```

Comment:
be good to mention the type here.  I guess it is ListControlAPLProps.

Reply:
Updated.